packages/rancher-monitoring/overlay/CHANGELOG.md

```diff
@@ -12,10 +12,8 @@ All notable changes from the upstream Prometheus Operator chart will be added to
-- Added `grafana-config-edit` and `grafana-config-view` default `ClusterRoles` to allow admins to assign roles to users to interact with Secrets or ConfigMaps utilized by Grafana. These can be enabled by setting `.Values.global.rbac.userRoles.create` (default: `true`). In a typical RBAC setup, you might want to assign the following users with these permissions:
-    - User who needs to be able to persist custom Grafana dashboards from the Grafana UI but does not need to be able to interact with Prometheus CRs: `grafana-config-edit` within the `.Values.grafana.sidecar.dashboards.searchNamespace` (default `grafana-dashboards`) namespace
-    - User who needs to be able to persist new Grafana datasources but does not need to be able to interact with Prometheus CRs: `.Values.grafana.sidecar.datasources.searchNamespace` (default `grafana-datasources`) namespace
+- Added `monitoring-admin`, `monitoring-edit`, and `monitoring-view` default `ClusterRoles` to allow admins to assign roles to users to interact with Prometheus Operator CRs. These can be enabled by setting `.Values.global.rbac.userRoles.create` (default: `true`). In a typical RBAC setup, you might want to use a `ClusterRoleBinding` to bind `monitoring-edit` or `monitoring-view` to a Subject to allow them to set up or view `ServiceMonitors` / `PodMonitors` / `PrometheusRules` and view `Prometheus` or `Alertmanager` CRs. If `.Values.global.rbac.userRoles.aggregateRolesForRBAC` is enabled, these ClusterRoles will aggregate into the respective default ClusterRoles provided by Kubernetes
+- Added `monitoring-config-edit` and `monitoring-config-view` default `ClusterRoles` to allow admins to assign roles to users to be able to edit / view `Secrets` and `ConfigMaps`. These can be enabled by setting `.Values.global.rbac.userRoles.create` (default: `true`). In a typical RBAC setup, you might want to use a `RoleBinding` to bind `monitoring-config-edit` or `monitoring-config-view` to a Subject within a specific namespace. A typical use case for this would be to bind `monitoring-config-edit` permissions to a Subject in the `cattle-monitoring-system` namespace to allow them to modify Secrets / ConfigMaps tied to the deployment, such as your Alertmanager Config Secret. Another typical use case for this would be to bind `monitoring-config-edit` permissions to a Subject in the `grafana-dashboards` namespace to allow them to create / delete ConfigMaps that allow the user to persist Grafana Dashboards on the cluster.
```

Comment:
Should this information be documented anywhere else?

Reply:
Can you send this to Catherine?